src/Core/Models/RegisteredSystem.cs

```diff
@@ -21,11 +22,13 @@ public class RegisteredSystem
         /// they should be aware of their own previous system names 
         /// when giving the new system it's id.
```

Comment:
wondering if the description, defaultrights must be marked as required

Reply:
it will probably only be visible on a GUI, and probably not needed in full API solution?

Reply:
I think input to that api does not differ based on the client

Reply:
i mean these information are mandarory

Reply:
ok, will mark it as req in new commit